www/includes/utility.php

```diff
@@ -893,6 +879,8 @@ function entities_to_numbers($string) {
 
 function make_member_url($name, $const = '', $house = HOUSE_TYPE_COMMONS, $pid = NULL) {
 
+    # convert name to ISO so we have sensible URLs
+    $name = iconv('UTF-8', 'ISO-8859-1//TRANSLIT', $name);
```

Comment:
This still needs sorting – is the issue that `$s2` contains ISO-8859-1 characters and needs updating for UTF-8? Really, we should be able to use `iconv("utf-8", "ascii//TRANSLIT", $name)` but it looks like iconv versions might cause issues there, so probably easier to stick with the search and replace if that covers everything.